docker-logs.txt

```diff
@@ -0,0 +1,1478 @@
+2025-01-02 16:15:20.389430: Logs for Test tests/bdd/features/volume/topology/test_node_topology.py::test_suitable_nodes_which_contain_volume_spread_topology_key[True-zone-us-1-<=-succeed-must be-zone-us=us-west-1,zone-us=different-value] (call):
```

Comment:
This file is not needed right?

Reply:
Thanks, seems like the gitignore was added after the  file had been commited so has to be removed. Done